apps/extension/src/application/trading-copilot/subscriptions-management/components/subscriptions-list/subscription-item.tsx

```diff
@@ -63,21 +63,21 @@ export const SubscriptionItem = ({ subscription, onRemove }: Properties) => {
       <div className="flex items-center">
         <LazyImage
           src={avatarQuery.data}
-          className="size-5 rounded-full"
+          className="size-7 rounded-full"
           fallbackComponent={
-            <Icon size={20} name="PersonIcon" className="rounded-full" />
+            <Icon size={24} name="PersonIcon" className="rounded-full text-neutral-700 bg-neutral-300 p-1" />
```

Comment:
to match the designs we should use a different icon with `size={20}` wrapped in 
`rounded-full bg-neutral-200 size-[32px]` div